packages/asset-service/src/service/AssetService.test.ts

```diff
@@ -145,9 +145,10 @@ describe('AssetService', () => {
     it('should return the overridden description if it exists', async () => {
       const assetService = new AssetService(assetFileUrl)
 
-      await expect(assetService.description({ asset: EthAsset })).resolves.toEqual(
-        'overriden description'
-      )
+      await expect(assetService.description({ asset: EthAsset })).resolves.toEqual({
+        description: 'overriden description',
```

Comment:
also needs to be fixed on line 41: https://github.com/shapeshift/lib/blob/3c88bd562a96b29409673bf3a7e760ec4582f725/packages/asset-service/src/service/AssetService.test.ts#L41
```suggestion
        description: 'overridden description',
```